components/builder-protocol/protocols/depotsrv.proto

```diff
@@ -11,10 +11,11 @@ message Package {
   required PackageIdent ident = 1;
   required string checksum = 2;
   required string manifest = 3;
-  repeated PackageIdent deps = 4;
-  repeated PackageIdent tdeps = 5;
-  repeated uint32 exposes = 6 [packed=true];
-  optional string config = 7;
+  required string target = 4;
+  repeated PackageIdent deps = 5;
+  repeated PackageIdent tdeps = 6;
+  repeated uint32 exposes = 7 [packed=true];
+  optional string config = 8;
```

Comment:
I might be way off about this, but I want to say that when you modify protobufs, you should add new fields to the end of the definition, so that the order of the existing fields is preserved.  I might be mis-remembering a conversation I had with @reset so it's possible this is no big deal.